src/luarocks/deps.lua

```diff
@@ -506,15 +506,13 @@ function deps.check_lua(vars)
    end
    local shortv = cfg.lua_version:gsub("%.", "")
    local libnames = {
+      "luajit-" .. cfg.lua_version,
       "lua" .. cfg.lua_version,
       "lua" .. shortv,
       "lua-" .. cfg.lua_version,
       "lua-" .. shortv,
       "lua",
    }
-   if cfg.luajit_version then
```

Comment:
Would it be possible to make this scenario detect `cfg.luajit_version` instead? I'm worried if adding luajit to the list inconditionally would make it pick up the luajit library when the intention is to use PUC Lua

Reply:
Note that `luajit_version` is typically missing in the config (#1042, #1043).